src/components/shared/PaymentBankData.vue

```diff
@@ -163,6 +162,10 @@ export default Vue.extend( {
 				);
 			}
 		},
+		setAccountId: function ( accountId: string ) {
+			this.$props.accountId = accountId;
```

Comment:
I think this should be `this.$data.accountId`, `this.$props` should never be set in a class and we don't have a property named like that.